PythonFDKIntroduction/README.md

```diff
@@ -1,58 +1,71 @@
 Python functions. It shows how easy it is to build, deploy and test
 functions written in Python.
 
 ## What is an FDK?
 
-The Fn Project allows developers to choose the intercommunication protocol format between an application and serverless function. 
-There are currently 3 different formats that Fn defines:
+The Fn Project allows developers to choose the intercommunication protocol
+format between an application and serverless function.  There are currently
+three different formats that Fn defines:
 
-    default: whatever an application sends inside the HTTP request body will be transferred to the function
-    HTTP: the function will receive the full HTTP request
-    JSON: the function will receive a JSON object very similar to CNCF OpenEvents
+* default: whatever an application sends inside the HTTP request body will be transferred to the function
+* HTTP: the function receives the full HTTP request
+* JSON: the function receives a JSON object very similar to CNCF OpenEvents
 
-What is the difference between these formats? Why are there three and not just one? 
-The short answer is that the default format is applied to cold functions only, 
-meaning the functions container lives no longer than the time needed to process a single request. 
-This means higher latencies due to container start/stop times, etc. 
-In a high-performance low-latency situation, this may not be acceptable. 
-In contrast, HTTP and JSON formats are applied to hot functions — meaning that the function stays alive 
-as long as there are more requests to process, but not longer than the time defined by idle_timeout while waiting for more work). 
-The HTTP format makes developers feel like they are talking to a web server because it looks very simple. 
-JSON format is a bit different because Fn automatically performs the steps necessary to assemble a JSON object from the HTTP request.
+What is the difference between these formats? Why are there three and not just
+one?
 
-So, what is an FDK and why do we need it? No matter what kind of format is defined for a particular function, 
-Fn will serve the requests through STDIN and wait on STDOUT for the response and STDERR for function logs. 
-In order to make a developer’s life simpler, the Fn team developed a set of libraries for different programming languages like Java, Python, Node, and Go. 
-The main goal of the FDK is to let developers focus on their functions and hide all routine complexities underneath.
+The short answer is the default format is applied only to cold functions.
+A cold function container lives no longer than the time needed to process
+a single request.  This means higher latencies due to container start/stop
+times, etc.  In a high performance, low latency situation this may not be
+acceptable.
 
-Please note that FDK-Python supports only JSON format.
+In contrast, HTTP and JSON formats are applied to hot functions. A hot function
```

Comment:
In contrast, CloudEvent and JSON formats are applied to hot functions.

HTTP format no longer supported by fdk-python